ert_shared/ensemble_evaluator/ensemble/builder.py

```diff
@@ -79,18 +80,33 @@ def build(self):
         return super().build()
 
 
+class Transformation(str, Enum):
+    TAR = "TAR_SUMMARY"
+    SUMMARY = "SUMMARY_VECTOR"
+
+
 class _FileIO(_IO):
-    def __init__(self, name: str, path: Path, mime: str) -> None:
+    def __init__(
+        self,
+        name: str,
+        path: Path,
+        mime: str,
+        transformations: List[Transformation] = [],
```

Comment:
Wouldn't it make sense to put it on the `_IO` base instead? We probably want transformation on all io.

Reply:
If that was the case (to have it for all io), I can do it of-course...